Public/Src/Cache/MemoizationStore/Library/Service/LocalCacheServer.cs

```diff
@@ -42,15 +42,9 @@ public class LocalCacheServer : LocalContentServerBase<ICache, ICacheSession>
         : base(logger, fileSystem, scenario, cacheFactory, localContentServerConfiguration)
         {
             var storesByName = new Dictionary<string, IContentStore>();
-            foreach (var kvp in localContentServerConfiguration.NamedCacheRoots)
+            foreach (var kvp in StoresByName)
             {
-                AbsolutePath cacheRootPath = kvp.Value;
-                fileSystem.CreateDirectory(cacheRootPath);
-
-                var cache = cacheFactory(cacheRootPath);
-                Contract.Assert(cache is IContentStore, $"Attempted to setup a cache named '{kvp.Key}' that is not an {nameof(IContentStore)} at path {cacheRootPath}, type used is {cache.GetType().Name}");
-
-                storesByName.Add(kvp.Key, (IContentStore)cache);
+                storesByName.Add(kvp.Key, (IContentStore)kvp.Value);

```

Comment:
Very much dislike use of local variable with same name as property; caused a good 1-2 minutes of confusion.

Reply:
Renamed to a better suited name.

---
In reply to: [321938875](https://github.com/microsoft/BuildXL/pull/845#discussion_r321938875) [](ancestors = 321938875)